### PR TITLE
Add factory girl

### PR DESCRIPTION
**Pull request by Bigless27**

### CODE REVIEW

File: spec/factory.rb [hunk 1]
```diff
@@ -0,0 +1,55 @@
+FactoryGirl.define do
+
+
+  factory :user do
+    username { Faker::Internet.user_name  }
+    password 'password'
```
Comment (dijonkitchen) [inline spec/factory.rb]: Should this be password_digest?

File: spec/factories.rb [hunk 2]
```diff
@@ -0,0 +1,53 @@
+
+FactoryGirl.define do
+
+
+  factory :user do
+    username { Faker::Internet.user_name  }
+    password 'password'
```
Comment (dijonkitchen) [inline spec/factories.rb]: Should this be password_digest?

File: spec/factories.rb [hunk 3]
```diff
@@ -0,0 +1,53 @@
+
+FactoryGirl.define do
+
```
Comment (dijonkitchen) [inline spec/factories.rb]: Kill extra lines

File: spec/factories.rb (hunk 2 above)
Comment (Bigless27) [inline spec/factories.rb]: Rails knows to associate Password with Password_digest in the user model

File: spec/models/answer_spec.rb [hunk 4]
```diff
@@ -1,5 +1,15 @@
...
+
 RSpec.describe Answer, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+
+  before(:each) do
+    @comment= FactoryGirl.create(:comment_answer)
+  end
+
+  before(:each) do
+    @vote= FactoryGirl.create(:vote_answer)
```
Comment (dijonkitchen) [inline spec/models/answer_spec.rb]: Combine and fix equal spacing

File: spec/factories.rb [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+    title {Faker::Book.title}
+    body {Faker::Team.name}
+    association :user
+  end
+
+  factory :comment_question, class: "Comment" do
+    association :commentable, factory: :question
+    response {Faker::Hipster.sentence}
+    association :user
+
```
Comment (dijonkitchen) [inline spec/factories.rb]: Kill extra lines

File: spec/models/comment_spec.rb [hunk 6]
```diff
@@ -1,5 +1,13 @@
 require 'rails_helper'
 
 RSpec.describe Comment, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+
+  describe 'polymorphism' do
+    it 'Has polmorphic associations' do
+      ryan= User.new(username: "bigless", password: "hello")
```
Comment (dijonkitchen) [inline spec/models/comment_spec.rb]: Let's use FactoryGirl for this

File: spec/spec_helper.rb [hunk 7]
```diff
@@ -21,6 +22,11 @@
 #
 # See http://rubydoc.info/gems/rspec-core/RSpec/Core/Configuration
 RSpec.configure do |config|
+
+ config.before(:all) do
```
Comment (dijonkitchen) [inline spec/spec_helper.rb]: What's this do?

**Comment by dijonkitchen** [comment]
:+1:

**Pull request merged**